lighthouse-core/gather/drivers/driver.js

```diff
@@ -489,11 +489,22 @@ class Driver {
     });
   }
 
-  beginEmulation() {
-    return Promise.all([
-      emulation.enableNexus5X(this),
-      emulation.enableNetworkThrottling(this)
-    ]);
+  beginEmulation(flags) {
+    let emulations = [];
```

Comment:
Reason why I don't use `const` here is because I modify the object. When using const I assume the data is not going to be changed. But I can change it :)